app/scripts/modules/core/src/pipeline/config/stages/wait/SkipWait.tsx

```diff
@@ -49,8 +52,20 @@ export class SkipWait extends React.Component<ISkipWaitProps, ISkipWaitState> {
     });
   };
 
+  public componentWillReceiveProps() {
+    this.runningTime.checkStatus();
```

Comment:
does this need a null check for `this.runningTime`?

Reply:
also, this isn't super actionable right now but at some point we will need to move things like this out of `componentWillReceiveProps` given that it'll be deprecated in React 17

Reply:
yeap, good catch